### ISSUE TITLE
Side menu no longer highlights selected page

### ISSUE THREAD

**Issue by rydmike** [invertase/docs.page#385]
The side menu no longer highlights and shows the selected page. It used to do this, imo this feature is critical. 

Is there a new way to enable it or is it just gone or broken?

Examples:

- [Docs.page](https://use.docs.page/configuration#anchors)

![Screenshot 2024-10-06 at 17 20 00](https://github.com/user-attachments/assets/792cf45e-234c-4d51-85db-3499353f3905)

- [FlexColorScheme](https://docs.flexcolorscheme.com/api_guide)

![Screenshot 2024-10-06 at 17 11 44](https://github.com/user-attachments/assets/6cb3d05c-a6d0-48e0-b7f7-3b29b15d806c)

**Comment by Ehesp** [comment]
Looks like it's broken on domain based docs, will fix tomorrow 👍

**Comment by Ehesp** [comment]
Fixed this, there's just one small issue left with "index" pages not showing - will investigate!

**Comment by rydmike** [comment]
Yes was about to say the same, works again, except for the index page.

**Comment by alexdukeinvertase** [comment]
Hey, this has been fixed as part of our V2 release. Please re-open if you see it again.
